hasjob/views/index.py

```diff
@@ -141,20 +143,22 @@ def fetch_jobposts(request_args, request_values, is_index, board, board_jobs, gk
         data_filters['anywhere'] = True
         # Only works as a positive filter: you can't search for jobs that are NOT anywhere
         basequery = remote_location_query
-    if 'currency' in request_args and request_args['currency'] in CURRENCY.keys():
-        currency = request_args['currency']
+
+    currency = filters.get('currency') or request_args.get('currency')
+    if currency in CURRENCY.keys():
         data_filters['currency'] = currency
         basequery = basequery.filter(JobPost.pay_currency == currency)
         pay_graph = currency
     else:
         pay_graph = False
-    if getbool(request_args.get('equity')):
+    if getbool(filters.get('equity') or request_args.get('equity')):
         # Only works as a positive filter: you can't search for jobs that DON'T pay in equity
         data_filters['equity'] = True
         basequery = basequery.filter(JobPost.pay_equity_min != None)  # NOQA
-    if 'pay' in request_args or ('pmin' in request_args and 'pmax' in request_args):
-        if 'pay' in request_args:
-            f_pay = string_to_number(request_args['pay'])
+
+    if filters.get('pay') or 'pay' in request_args or ('pmin' in request_args and 'pmax' in request_args):
+        if 'pay' in request_args or filters.get('pay'):
+            f_pay = filters['pay'] if filters.get('pay') else string_to_number(request_args['pay'])
```

Comment:
Also missing tags and domains here.

Reply:
Still missing tags and domains here.

Reply:
I wanted to add them when they get used in the UI. Would you like me to add them now?

Reply:
Yes. Our first use case is to replace boards like cto.hasjob.co with a filterset like hasjob.co/f/cto.

Reply:
Just like `archive` is accessible through the URL and not through the UI, these filters can also be URL-only until we work out decent UI for them.